Maya/tools/mathUtils.py

```diff
@@ -305,7 +305,7 @@ def sortByDistance(nodes, point, points):
     :return: Sorted node list and sorted points list
     :rtype: tuple(list, list)
     """
-    iter = zip(nodes, points)
+    iter = list(zip(nodes, points))

```

Comment:
This indeed changed since Python 3 (.6?) and needs to be fixed.

I would suggest to use sorted() on the iterator as opposed to casting it to a list(), because it gets cast again when unzipping. For example:

```
iter = zip(nodes, points)
iter = sorted(iter, key=lambda x: (x[1] - point).length())
```

This may be pedantic but I think it will deal better with larger datasets. By giving an iterable to sorted() we leave the potential for optimization to the Python interpreter. Casting to a list forces a large memory allocation. I doubt that Python is smart enough to see the `zip(*iter)` directly after, but in theory it could end up allocating just the return values and filling them in a sorted manner.